airflow/api_fastapi/execution_api/routes/xcoms.py

```diff
@@ -147,7 +147,7 @@ def get_xcom(
         )
 
     try:
-        xcom_value = BaseXCom.orm_deserialize_value(result)
+        BaseXCom.orm_deserialize_value(result)
```

Comment:
Is this whole block needed now? Wouldn't we get that anyway from creating the `XComResponse` object?

Reply:
We would definitely. I think it was added in the past to check for any "wrong" data in the xcom table so that we dont have to deal with the deser issues in the task runner and it can be caught in the API server directly. 

I can remove it, but WDYT about that reasoning?

Reply:
SDK client pretty much has to deal with xcom deser errors, because a custom xcom backend could just as easily return something invalid. (And it should always be the job of a client to validate what it's given, it should not trust a third party to send it valid data.)

Reply:
Thanks. I will remove that block